README.md

```diff
@@ -14,6 +14,9 @@ Python is a dev dependency. It's not needed to run the docker containers, but ne
   One or both of the following, depending on if you want to run the stable build `latest` or `dev`:
     - a. `docker pull obolibrary/odkfull:latest`
     - b. `docker pull obolibrary/odkfull:dev`
+4. Optional: [`odkrunner`](https://github.com/gouttegd/odkrunner): Alternative to `run.sh`. If using this, you would run
+    like `odkrun make build-mondo-ingest` instead of `sh run.sh make build-mondo-ingest`. Suggestion: rename the binary 
+    to `odk`; this is the convention of `mondo-ingest` developers. 
```

Comment:
I would leave this as odkrun for now just to more easily match the odkrunner repo docs.